### PR TITLE
fix: Add exportparts to mux-player so we can style nested video

### PR DESCRIPTION
**Pull request by luwes**
Original at https://github.com/muxinc/elements/pull/384 by @mikker

> This is needed to allow adding CSS to a nested web component via the `part()` syntax.
> 
> See [developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/exportparts](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/exportparts)
> 
> Tested and it works.
> 
> ```
> exportparts="video: video"
> ```
> 
> ```css
> mux-player::part(video) {
>   border: 3px solid red;
> }
> ```

### CODE REVIEW

File: packages/mux-player/src/template.ts [hunk 1]
```diff
@@ -105,6 +105,7 @@ export const content = (props: MuxTemplateProps) => html`
           : false
       }"
       cast-stream-type="${isLive(props) ? 'live' : false}"
+      exportparts="video"
```
Comment (gkatsev) [inline packages/mux-player/src/template.ts:108]: is the reason why it's only needed here because it's not in the shadow dom of mux-player?

Comment (luwes) [reply packages/mux-player/src/template.ts:108]: yup, CSS parts need to be forwarded between multiple shadom doms

**Review by gkatsev — APPROVED** [review]
My question isn't blocking as I confirmed it's working.

File: packages/mux-player/src/template.ts (hunk 1 above)
Comment (luwes) [reply packages/mux-player/src/template.ts:108]: I wish we didn't have to add such big attributes for themes but yea 🤷‍♂️

**Comment by mikker** [comment]
Glorious! Thank you. Love Mux ❤️🧡💛💚💙💜

**Comment by luwes** [comment]
@mikker thank you for the contribution! ❤️

**Pull request merged**